modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -286,32 +286,44 @@ module LocaleModel {
   private extern proc chpl_memhook_md_num(): chpl_mem_descInt_t;
 
   // The allocator pragma is used by scalar replacement.
+
+  // Note that there's 2 chpl_here_alloc() procs. This one explicitly takes an
```

Comment:
Grammar police: "there are 2", or "there're" (except yuk). But not "there is 2" or "there's 2".